pkg/render/intrusion_detection.go

```diff
@@ -1404,18 +1409,25 @@ func (c *intrusionDetectionComponent) adPersistentVolumeClaim() *corev1.Persiste
 
 func (c *intrusionDetectionComponent) adAPIDeployment(configureADStorage bool) *appsv1.Deployment {
```

Comment:
Looks like this func no longer needs the bool passed in, since there is a new method.

Reply:
switched to using the method call instead https://github.com/tigera/operator/pull/2300/commits/27a659534b6ddf2f6390379f8f3f515f4721574a